### ISSUE TITLE
hosting multi day event

### ISSUE THREAD

**Issue by amanintech** [commudle/commudle-ng#25]
Currently, when hosting an event it gives the option of a single from date.
![image](https://user-images.githubusercontent.com/28403318/84480960-fdc62980-acb2-11ea-8231-606eeac448ce.png)
We should add a much convenient way of choosing multi-day events.

**Comment by arpansac** [comment]
Yup, initially we had it, then I changed it to just one day because this was one of the rare use cases. 
We need it!

**Comment by arpansac** [comment]
This is live completely for quite some time. Thank you so much. 
Never got back to the issues; now getting more active.
